### PR TITLE
Test suma connection UI

### PR DESCRIPTION
**Pull request by nelsonkopliku**
# Description

This PR adds the ability to test the connection with the software updates provider: SUMA.
Uses endpoint implemented in https://github.com/trento-project/web/pull/2444

Test connection button is enabled only if there are settings saved and if there is no test currently in progress.

Notifications are shown in both success or failure cases.

https://2453.prenv.trento.suse.com/

## How was this tested?

Automated tests.

### CODE REVIEW

File: assets/js/pages/SettingsPage/SettingsPage.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Transition } from '@headlessui/react';
+import _ from 'lodash';
```
Comment (dottorblaster) [inline assets/js/pages/SettingsPage/SettingsPage.jsx:4]:
```suggestion
import { values, isUndefined } from 'lodash';
```

Comment (jamie-suse) [inline assets/js/pages/SettingsPage/SettingsPage.jsx:4]: I see in the rest of the codebase we use this import style

```suggestion
import { isUndefined } from 'lodash';
```

File: assets/js/pages/SettingsPage/SettingsPage.jsx [hunk 2]
```diff
@@ -81,12 +83,17 @@ function SettingsPage() {
...
     editing: editingSoftwareUpdatesSettings,
+    testingConnection: testingSoftwareUpdatesConnection,
   } = useSelector(getSoftwareUpdatesSettings);
 
   const suseManagerValidationErrors = useSelector(
     getSoftwareUpdatesSettingsErrors
   );
 
+  const hasSoftwareUpdatesSettings = _.values(settings).every(
+    (it) => !_.isUndefined(it)
```
Comment (dottorblaster) [inline assets/js/pages/SettingsPage/SettingsPage.jsx:94]:
```suggestion
    (value) => !_.isUndefined(value)
```

**Review by dottorblaster — CHANGES_REQUESTED** [review]
Just a couple changes to apply then we can merge :+1:

**Review by jamie-suse — APPROVED** [review]
:rocket:

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**